components/calendar/appearance.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Appearance
+page_title: Calendar Appearance
+description: Appearance settings of the Calendar for Blazor.
+slug: calendar-appearance
+tags: telerik,blazor,calendar,appearance
+published: True
+position: 30
+---
+
+# Appearance Settings
+
+You can control the appearance of the Calendar by setting the following attribute:
```

Comment:
Try to make the first sentence sound a bit more introductory and detached from the article structure. This will make it future-proof as well.

```suggestion
This article outlines the available Calendar parameters, which control its appearance.
```